Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN /bin/bash -e /security_updates.sh && \
     apt-get install -yqq --no-install-recommends \
         nginx-light \
         ca-certificates \
+        libnginx-mod-brotli \
```

Comment:
does it matter that we do not ask for a specific version here? or do we want to pin to a specific version and then tag the docker image with the same?

Reply:
@kangman thank you that's a good question. I'm not sure the we need to, but if we will, should we also specify version for `nginx-light` so we can control the pair? @ingluisjimenez what do you think?

Reply:
Fine with either way, but if we can tag then that seems more controller and a better idea

Reply:
@kangman @ingluisjimenez what would be the right way to specify the version? I've tried several things but seems i still missing something. For these options:

``` nginx-light=1.20.1-2+ubuntu20.04.1+deb.sury.org+1```
``` nginx-light-1.20.1-2+ubuntu20.04.1+deb.sury.org+1```
``` nginx-light=1.20.1```

I'm getting
```Version '1.20.1-1+ubuntu21.04.1+deb.sury.org+1' for 'nginx-light' was not found```

But seem the release exists:
https://launchpad.net/~ondrej/+archive/ubuntu/nginx/+build/21741614

Reply:
@ingluisjimenez @kangman could you please review the updated version.
For some reason I can't build it locally